files/k3s-install-agent.sh

```diff
@@ -12,32 +12,76 @@ do
 done
 }
 
-# Disable firewall 
-/usr/sbin/netfilter-persistent stop
-/usr/sbin/netfilter-persistent flush
-
-systemctl stop netfilter-persistent.service
-systemctl disable netfilter-persistent.service
-
-# END Disable firewall
+if test -f /etc/lsb-release; then
+  operating_system="ubuntu"
+else
+  operating_system="oraclelinux"
+fi
+
+if [[ "$operating_system" == "ubuntu" ]]; then
+  # Disable firewall 
+  /usr/sbin/netfilter-persistent stop
+  /usr/sbin/netfilter-persistent flush
+
+  systemctl stop netfilter-persistent.service
+  systemctl disable netfilter-persistent.service
+
+  # END Disable firewall
+
+  apt-get update
+  apt-get install -y software-properties-common jq
+  DEBIAN_FRONTEND=noninteractive apt-get upgrade -y
+  %{ if install_nginx_ingress }
+  DEBIAN_FRONTEND=noninteractive apt-get install --no-install-recommends -y python3 python3-pip nginx
+  systemctl enable nginx
+  pip install oci-cli
+  %{ endif }
+fi
+
+if [[ "$operating_system" == "oraclelinux" ]]; then
+  # Disable firewall
+  systemctl disable --now firewalld
+  # END Disable firewall
+
+  # Fix iptables/SELinux bug
+  echo '(allow iptables_t cgroup_t (dir (ioctl)))' > /root/local_iptables.cil
+  semodule -i /root/local_iptables.cil
+
+  dnf -y update
+  dnf -y install jq curl
+  %{ if install_nginx_ingress }
+  if grep -q "el9" /etc/os-release; then
+    dnf -y install python39-oci-cli python3-jinja2 nginx-all-modules
+  else
+    dnf -y module enable nginx:1.20 python36:3.6
+    dnf -y install python36-oci-cli python3-jinja2 nginx-all-modules
+  fi
```

Comment:
```suggestion
  if grep -q "el9" /etc/os-release; then
    dnf -y install oraclelinux-developer-release-el9
    dnf -y install python39-oci-cli python3-jinja2 nginx-all-modules
  else
    dnf -y install oraclelinux-developer-release-el8
    dnf -y module enable nginx:1.20 python36:3.6
    dnf -y install python36-oci-cli python3-jinja2 nginx-all-modules
  fi
```

Reply:
The package seems to be already installed:

```
dnf list installed | grep oraclelinux-developer-release-
oraclelinux-developer-release-el8.aarch64 1.0-7.el8                                   @ol8_baseos_latest
```
but for safety I will try to install the package